### PR TITLE
Created note listing page UI

### PR DESCRIPTION
**Pull request by sunnyshahabuddin**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            style="body3"
+          >
+            Created {calculateCreatedAgo(note.created_at)}
+          </Typography>
+        </Tooltip>
+        <Avatar
+          user={{
+            imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
+          }}
+          onClick={function noRefCheck() {}}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: Go through the Slack thread very carefully and fix the issue here.

**Review by yedhink — COMMENTED** [review]
@sunnyshahabuddin

**Comment by yedhink** [comment]
@sunnyshahabuddin Also go through https://handbook.neetokb.com/articles/git-and-github-essentials very carefully and see if you need to make any changes in this PR>

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 1 above)
Comment (sunnyshahabuddin) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: I have removed the onClick function as it there was no action to be performed when the avatar is clicked. I will be careful to use the arrow function next time in accordance with the react best practices listed in the slack thread.

> @sunnyshahabuddin Also go through https://handbook.neetokb.com/articles/git-and-github-essentials very carefully and see if you need to make any changes in this PR>
I have also changed pull request title as per "git and github essentials". I will make sure to cross check following points before creating a pull request next time:

1. Title should always begin with a capital letter and not with a number.
2. Link the pull request with the associated issue.
3. Write the first comment mentioning you, and to ask for a review.

Please let me know for any other changes, or points that I missed.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: >Please let me know for any other changes, or points that I missed.

Sounds good. You haven't missed any points now. 👍

Comment (sunnyshahabuddin) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: Thank you Yedhin.

**Pull request merged**